### ISSUE TITLE
error running ./mjpc

### ISSUE THREAD

**Issue by lrchit** [google-deepmind/mujoco_mpc#345]
Hi,

Thanks for your incredible work of mujoco_mpc! I'm trying to use it on my Ubuntu 20.04, but after every thing went well like this:
![image](https://github.com/user-attachments/assets/7c171fac-ad2d-445e-9fff-8f532fb44115)
I ran cd /bin and ./mjpc and went across this:
![image](https://github.com/user-attachments/assets/e2f70a25-b017-4821-a57b-9baa26a040aa)
with mujoco screen:
![image](https://github.com/user-attachments/assets/3503f903-9f89-457e-84c5-25e0f248a7d6)

Could anyone tell me what happened there? Thanks in advance!

Best, 
Ruochen

**Comment by yuvaltassa** [comment]
Unless you are saying that you ran MJPC at HEAD, in which case we might have a model which does not comply with this requirement, in which case we have a bug 🙂 

cc @thowell

**Comment by thowell** [comment]
@lrchit which task produces this error?

**Comment by yuvaltassa** [comment]
@thowell I don't think OP reached the point where they even selected a task. 
IIUC this just happened to them at the launch of the app?

**Comment by lrchit** [comment]
> @thowell I don't think OP reached the point where they even selected a task. IIUC this just happened to them at the launch of the app?

Yes, I just ran ./mjpc and the error message appeared. Should the tasks be specified after ./mjpc?

**Comment by husnoo** [comment]
> > @thowell I don't think OP reached the point where they even selected a task. IIUC this just happened to them at the launch of the app?
> 
> Yes, I just ran ./mjpc and the error message appeared. Should the tasks be specified after ./mjpc?

Did you figure this out?

**Comment by asd417** [comment]
Running into same issue as well. Compiled myself on Linux 24.04 LTS with a separate mujoco 3.2.3 library

**Comment by asd417** [comment]
Figured out.

The reason I was getting this issue is because I was compiling ``libmujoco.so.3.2.3`` from the official mujoco 3.2.3 repo and using ``LD_LIBRARY_PATH`` to the compiled library.

Instead, I compiled the ``libmujoco.so.3.2.3`` using the mujoco-src provided in this repo at ``build/_deps/mujoco-src``
